### PR TITLE
[ACL] #107 - Fix Acls voting

### PR DESCRIPTION
**Pull request by crouillon**
Fix some Acls issues and allow to apply rights on AbstractClassContent

### CODE REVIEW

File: Rest/Controller/PageController.php [hunk 1]
```diff
@@ -199,7 +199,7 @@ public function getAction(Page $page)
     public function postAction(Layout $layout, Request $request, Page $parent = null)
     {
         if (null !== $parent) {
-            $this->granted(MaskBuilder::MASK_EDIT, $parent);
+            $this->granted('EDIT', $parent);
```
Comment (mickaelandrieu) [inline Rest/Controller/PageController.php]: :+1:

**Comment by mickaelandrieu** [comment]
Some tests are broken... probablu because the related tests are bad / to the correct expected behavior.

Would you mind to fix the related tests ? I'll merge ASAP

**Comment by crouillon** [comment]
Yes i'm on it

**Comment by mickaelandrieu** [comment]
thank you @crouillon , nice work !

**Pull request merged**